src/Nri/Ui/Block/V5.elm

```diff
@@ -284,6 +284,7 @@ type BlankLength
     = SingleCharacter
     | ShortWordPhrase
     | LongWordPhrase
+    | CharacterCount Int
```

Comment:
This addition causes a major bump:

```
---- Nri.Ui.Block.V5 - MAJOR ----

    Changed:
      - type BlankLength  = SingleCharacter | ShortWordPhrase | LongWordPhrase
      + type BlankLength
            = SingleCharacter
            | ShortWordPhrase
            | LongWordPhrase
            | CharacterCount Basics.Int

```

This is one of the reasons why in most noredink-ui components, we don't expose types directly, and instead will do something like:

```
type Customization = ThemeA | ThemeB

themeA : Attribute msg
themeA =
    Attribute (\config -> {config | theme = ThemeA})
```

Reply:
If my hand is forced to do a major version update, I will just remove the `SingleCharacter`, `ShortWordPhrase`, and `LongWordPhrase` options to clarify the API